### PR TITLE
[FrameworkBundle][Workflow] fix guard expressions

### PR DESCRIPTION
**Pull request by destillat**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

Possible fix for https://github.com/symfony/symfony/pull/28007#issuecomment-406524200

### CODE REVIEW

File: src/Symfony/Component/Workflow/EventListener/GuardExpression.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     * GuardConfiguration constructor.
+     * @param Transition $transition
+     * @param string $expression
+     */
+    public function __construct(Transition $transition, string $expression)
+    {
+        $this->transition = $transition;
+        $this->expression = $expression;
+    }
+}
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Workflow/EventListener/GuardExpression.php]: missing EOL. Also please remove all docblocks in this class: they don't provide anything on top of the existing type declaration, and our policy is to not add them in this situation.

**Comment by fabpot** [comment]
@lyrixx Can you review this one? Thank you.

**Comment by nicolas-grekas** [comment]
friendly ping @lyrixx

**Comment by lyrixx** [comment]
Hello. Sorry for the very late reply. I will have a look tomorrow or after tomorrow. Thanks

**Comment by lyrixx** [comment]
Thanks a lot @destillat. I finished you work in #29137 and I rebased against 3.4.
I'm closing this issue in favor my PR

**Pull request closed without merging**